### PR TITLE
cpu count component added

### PR DESCRIPTION
**Pull request by FarajiA**
## JIRA:
See ticket here - https://jira.rax.io/browse/MNRVA-56

## Description:
Adds CPU Count component 

## Testing:
Open the /docs index.html and navigate to `cpu-count`

## Screenshots:
(if applicable)

### CODE REVIEW

File: docs/assets/scripts/hedwig-main.umd.js [hunk 1]
```diff
@@ -6574,6 +6581,45 @@
...
+       */
+
+
+      connectedCallback() {
+        var defaults = new Defaults();
+        var margin = this.dataset.margin || defaults.margin;
+        var height = (this.dataset.height || defaults.height) - margin.top - margin.bottom;
+        var width = (this.dataset.width || defaults.width) - margin.left - margin.right;
+        var lineColor = this.dataset.lineColor || defaults.lineColor;
+        var unit = this.dataset.unit || 'count';
```
Comment (FarajiA) [inline docs/assets/scripts/hedwig-main.umd.js]: Thinking we can have some mechanism in place to change the unit of measurement from percentage.

Comment (kyleondata) [reply docs/assets/scripts/hedwig-main.umd.js]: yea I have been thinking the same thing. I think we can use data-* attrs for that

File: rollup.config.js [hunk 2]
```diff
@@ -22,6 +22,19 @@ export default [
 			commonjs() // so Rollup can convert any dependencies to an ES module
 		]
 	},
+	{
+		input: 'index.js',
+		output: {
+			name: 'hedwig-docs',
+			file: pkg.docs,
```
Comment (kyleondata) [inline rollup.config.js]: Nice

**Comment by FarajiA** [comment]
This was merged with conflicts resolved.

**Pull request closed without merging**